scripts/dev_scripts/integration_tests.sh

```diff
@@ -33,7 +33,7 @@ then
     cp $RESOURCES/settings.xml $HOMEDIR/.m2/
 fi
 
-# Running Macaron without config files
+# # Running Macaron without config files
```

Comment:
Was this `#` intended?

Reply:
No it's not. I will remove it.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/153/commits/08991703ae373e11b772c8c5628b9ce8d24a0f8b